pkg/daemon/update.go

```diff
@@ -226,17 +229,27 @@ func (dn *Daemon) enableUnit(unit ignv2_2types.Unit) error {
 	}
 	// The originating file to link
 	servicePath := filepath.Join(pathSystemd, unit.Name)
-	return os.Symlink(servicePath, wantsPath)
+	err := os.Symlink(servicePath, wantsPath)
+	if err == nil {
```

Comment:
I've seen `if err := os.Symlink(servicePath, wantsPath); err == nil {...}` style as well.  Which is idiomatic?

Reply:
@nstielau I've only seen the above when there is not an else statement. But since we are adding an else with V2 level logging declaration to cover both cases (err is nil and err != nil) then declaration--if--else seemed more clear and readable.

Reply:
other places of the code in this file check the negative case first (!=)... I have seen other golang code do the same thing.

Reply:
@rphillips I'll make this consistent. Good catch. Thanks!

Reply:
Done.